crates/core_arch/src/x86/sse2.rs

```diff
@@ -1752,7 +1806,7 @@ pub unsafe fn _mm_max_sd(a: __m128d, b: __m128d) -> __m128d {
 #[cfg_attr(test, assert_instr(maxpd))]
 #[stable(feature = "simd_x86", since = "1.27.0")]
 pub unsafe fn _mm_max_pd(a: __m128d, b: __m128d) -> __m128d {
-    maxpd(a, b)
+    simd_fmax(a, b)
```

Comment:
Are you sure that this change is correct with regards to edge cases like NaN and negative zero?

Reply:
No, I am not sure. I have dropped the commit.